disgordrouter/context.go

```diff
@@ -4,7 +4,7 @@ import (
 	"fmt"
 	"sync"
 
-	"github.com/andersfylling/snowflake"
+	"github.com/andersfylling/snowflake/v3"
```

Comment:
`Anders  Today at 9:43 AM`
```
remove andersfylling/snowflake
and use disgord.Snowflake
then you might be able to get around the module issue. not sure tho
the PR looks good. I'm just not officially supporting anything that is not a module
```

Reply:
These are my first PR I have done. Do I have to redo it after I made the change and commit it to my fork?

Reply:
No, you can keep updating your branch until I merge it

Reply:
> These are my first PR I have done. Do I have to redo it after I made the change and commit it to my fork?

Nope. It get's automatically added to this PR. You should see a new change below this conversation: "Changed to use disgord.snowflake"